server/opendp_apps/cypress_utils/management/commands/load_demo_data.py

```diff
@@ -0,0 +1,300 @@
+                                                 first_name='DP',
+                                                 last_name='Analyst',
+                                                 email='test_analyst@opendp.org')
+        analyst_user.set_password(analyst_password)
+        analyst_user.save()
+        self.write_success_msg(f'Analyst created: {analyst_user}')
+
+        # "Verify" users via email
+        #
+        VerifyModel = apps.get_model(app_label='account', model_name='emailaddress')
+        verify_analyst = VerifyModel(user=analyst_user,
+                                     email=analyst_user.email,
+                                     primary=True,
+                                     verified=True)
+        verify_analyst.save()
+
+        verify_depositor =  VerifyModel(user=depositor_user,
+                                        email=depositor_user.email,
+                                        primary=True,
+                                        verified=True)
+        verify_depositor.save()
+
+        # Create UploadFileInfo
+        #
+        depositor_setup = self.get_depositor_setup_info(analyst_user)
+        self.write_success_msg(f'DepositorSetupInfo created: {depositor_setup}')
+
+        upload_file = UploadFileInfo.objects.create(
+                    name='Teacher Survey',
+                    creator=analyst_user,
+                    data_profile=self.get_data_profile(),
+                    profile_variables=self.get_data_profile(),
+                    depositor_setup_info=depositor_setup,
+                    )
+        upload_file.save()
+        self.write_success_msg(f'UploadFileInfo created: {upload_file}')
+
+        # Add File to UploadFileInfo
+        #
+        filename = 'teacher_survey.csv'
+        filepath = join(DEMO_FILE_DIR, filename)
+        if not isfile(filepath):
+            user_msg = f'Failed to find test data file: {filename}'
+            self.stdout.write(self.style.ERROR(user_msg))
+            return
+
+        django_file = File(open(filepath, 'rb'))
+        upload_file.source_file.save(filename, django_file)
+        upload_file.save()
+        self.write_success_msg(f'Data file attached to UploadFileInfo')
+
+        # Add AnalysisPlan
+        #
+        plan = AnalysisPlan(name=f'Plan {get_rand_alphanumeric(7)}',
+                            analyst=analyst_user,
+                            dataset=upload_file,
+                            variable_info=depositor_setup.variable_info,
+                            user_step=AnalysisPlan.AnalystSteps.STEP_0700_VARIABLES_CONFIRMED)
+
+        plan.save()
+        self.write_success_msg(f'AnalysisPlan created: {plan}')
+        self.write_success_msg(f'>> Success! Process complete.', indent=False)
+
+    def is_demo_data_already_loaded(self) -> bool:
+        """Check if any of the demo data is already in the system"""
+        if OpenDPUser.objects.filter(username=self.depositor_username).first() is not None:
```

Comment:
Better to use .exists() for checks like this

Reply:
updated